tests/MongoDB.Driver.Core.Tests/Core/Configuration/ConnectionSettingsTests.cs

```diff
@@ -124,7 +134,9 @@ public void constructor_with_compressors_should_initialize_instance()
             var subject = new ConnectionSettings(compressors: compressors);
 
             subject.ApplicationName.Should().BeNull();
-            subject.Authenticators.Should().Equal(__defaults.Authenticators);
+            subject.AuthenticatorFactories.Should().NotBeNull();
+            var authenticators = subject.AuthenticatorFactories.Select(a => a.Create());
+            authenticators.Should().BeEmpty();
```

Comment:
Replace 137-139 with:

            subject.AuthenticatorFactories.Should().BeEmpty();